migrations/versions/201603171003_3d672dc5ee53_unify_lecture_style_ids.py

```diff
@@ -0,0 +1,50 @@
+"""Unify lecture style ids
+
+Revision ID: 3d672dc5ee53
+Revises: 29232c09e58a
```

Comment:
Also update the filename so it's in the right order (I'd simply go for whatever the current time is when you rebase and update this file)